vllm/worker/openvino_model_runner.py

```diff
@@ -72,9 +72,7 @@ def __init__(
         self.block_size = cache_config.block_size
 
         self.attn_backend = get_attn_backend(
-            self.model_config.get_num_attention_heads(self.parallel_config),
             self.model_config.get_head_size(),
-            self.model_config.get_num_kv_heads(self.parallel_config),
             self.model_config.get_sliding_window(),
             self.model_config.dtype,
             self.kv_cache_dtype,
```

Comment:
Should you be setting self.model_config.is_attention_free here too?

Reply:
Yeah, this is kind of the reason I didn't want to default `is_attention_free` to False TBH